//------------------------------------------------------------------------------
// <auto-generated>
//     This code was generated by a tool.
//
//     Changes to this file may cause incorrect behavior and will be lost if
//     the code is regenerated.
// </auto-generated>
//------------------------------------------------------------------------------

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using NUnit.Framework;

namespace Jint.Test262.Ch12
{
    [TestFixture]
    public class _12_10Fixture : Test262Fixture
    {
        public _12_10Fixture()
            : base("ch12\\12.10")
        {
        }

        [Test(Description = "with does not change declaration scope - vars in with are visible outside")]
        public void _12_10_0_1()
        {
            RunFile("12.10-0-1.js");
        }

        [Test(Description = "with introduces scope - name lookup finds function parameter")]
        public void _12_10_0_10()
        {
            RunFile("12.10-0-10.js");
        }

        [Test(Description = "with introduces scope - name lookup finds inner variable")]
        public void _12_10_0_11()
        {
            RunFile("12.10-0-11.js");
        }

        [Test(Description = "with introduces scope - name lookup finds property")]
        public void _12_10_0_12()
        {
            RunFile("12.10-0-12.js");
        }

        [Test(Description = "with introduces scope - that is captured by function expression")]
        public void _12_10_0_3()
        {
            RunFile("12.10-0-3.js");
        }

        [Test(Description = "with introduces scope - scope removed when exiting with statement")]
        public void _12_10_0_7()
        {
            RunFile("12.10-0-7.js");
        }

        [Test(Description = "with introduces scope - var initializer sets like named property")]
        public void _12_10_0_8()
        {
            RunFile("12.10-0-8.js");
        }

        [Test(Description = "with introduces scope - name lookup finds outer variable")]
        public void _12_10_0_9()
        {
            RunFile("12.10-0-9.js");
        }

        [Test(Description = "with - expression being Number")]
        public void _12_10_2_1()
        {
            RunFile("12.10-2-1.js");
        }

        [Test(Description = "with - expression being Boolean")]
        public void _12_10_2_2()
        {
            RunFile("12.10-2-2.js");
        }

        [Test(Description = "with - expression being string")]
        public void _12_10_2_3()
        {
            RunFile("12.10-2-3.js");
        }

        [Test(Description = "with introduces scope - restores the earlier environment on exit")]
        public void _12_10_7_1()
        {
            RunFile("12.10-7-1.js");
        }

        [Test(Description = "Using interation statement within \"with\" statement leading to normal completion")]
        public void S12_10_A1_10_T1()
        {
            RunFile("S12.10_A1.10_T1.js");
        }

        [Test(Description = "Using iteration statement within \"with\" statement leading to completion by exception")]
        public void S12_10_A1_10_T2()
        {
            RunFile("S12.10_A1.10_T2.js");
        }

        [Test(Description = "Using iteration statment withing \"with\" statement leading to completion by exception")]
        public void S12_10_A1_10_T3()
        {
            RunFile("S12.10_A1.10_T3.js");
        }

        [Test(Description = "Using iteration statement witthin \"with\" staement leading to completion by break")]
        public void S12_10_A1_10_T4()
        {
            RunFile("S12.10_A1.10_T4.js");
        }

        [Test(Description = "Using iteration statement within \"with\" statement leading to completion by break")]
        public void S12_10_A1_10_T5()
        {
            RunFile("S12.10_A1.10_T5.js");
        }

        [Test(Description = "Calling a function within \"with\" statement declared without the statement, leading to normal completion")]
        public void S12_10_A1_11_T1()
        {
            RunFile("S12.10_A1.11_T1.js");
        }

        [Test(Description = "Calling a function within \"with\" statement declared without the statement, leading to normal completion by \"return\"")]
        public void S12_10_A1_11_T2()
        {
            RunFile("S12.10_A1.11_T2.js");
        }

        [Test(Description = "Calling a function within \"with\" statement declared without the statement, leading to normal completion by \"return\"")]
        public void S12_10_A1_11_T3()
        {
            RunFile("S12.10_A1.11_T3.js");
        }

        [Test(Description = "Calling a function within \"with\" statement declared without the statement, leading to completion by exception")]
        public void S12_10_A1_11_T4()
        {
            RunFile("S12.10_A1.11_T4.js");
        }

        [Test(Description = "Calling a function within \"with\" statement declared without the statement, leading to completion by exception")]
        public void S12_10_A1_11_T5()
        {
            RunFile("S12.10_A1.11_T5.js");
        }

        [Test(Description = "Calling a function without \"with\" statement declared within the statement, leading to normal completion")]
        public void S12_10_A1_12_T1()
        {
            RunFile("S12.10_A1.12_T1.js");
        }

        [Test(Description = "Calling a function without \"with\" statement declared within the statement, leading to normal completion by \"return\"")]
        public void S12_10_A1_12_T2()
        {
            RunFile("S12.10_A1.12_T2.js");
        }

        [Test(Description = "Calling a function without \"with\" statement declared within the statement, leading to normal completion by \"return\"")]
        public void S12_10_A1_12_T3()
        {
            RunFile("S12.10_A1.12_T3.js");
        }

        [Test(Description = "Calling a function without \"with\" statement declared within the statement, leading to completion by exception")]
        public void S12_10_A1_12_T4()
        {
            RunFile("S12.10_A1.12_T4.js");
        }

        [Test(Description = "Calling a function without \"with\" statement declared within the statement, leading to completion by exception")]
        public void S12_10_A1_12_T5()
        {
            RunFile("S12.10_A1.12_T5.js");
        }

        [Test(Description = "Using \"with\" inside of global context leading to normal completion")]
        public void S12_10_A1_1_T1()
        {
            RunFile("S12.10_A1.1_T1.js");
        }

        [Test(Description = "Using \"with\" inside of global context leading to completion by exception")]
        public void S12_10_A1_1_T2()
        {
            RunFile("S12.10_A1.1_T2.js");
        }

        [Test(Description = "Using \"with\" inside of global context leading to completion by exception")]
        public void S12_10_A1_1_T3()
        {
            RunFile("S12.10_A1.1_T3.js");
        }

        [Test(Description = "Calling a function without \"with\" statement when the statement itself is declared within the function declaration, leading to normal completion")]
        public void S12_10_A1_2_T1()
        {
            RunFile("S12.10_A1.2_T1.js");
        }

        [Test(Description = "Calling a function without \"with\" statement when the statement itself is declared within the function declaration, leading to normal completion by \"return\"")]
        public void S12_10_A1_2_T2()
        {
            RunFile("S12.10_A1.2_T2.js");
        }

        [Test(Description = "Calling a function without \"with\" statement when the statement itself is declared within the function declaration, leading to normal completion by \"return\"")]
        public void S12_10_A1_2_T3()
        {
            RunFile("S12.10_A1.2_T3.js");
        }

        [Test(Description = "Calling a function without \"with\" statement when the statement itself is declared within the function declaration, leading to completion by exception")]
        public void S12_10_A1_2_T4()
        {
            RunFile("S12.10_A1.2_T4.js");
        }

        [Test(Description = "Calling a function without \"with\" statement when the statement itself is declared within the function declaration, leading to completion by exception")]
        public void S12_10_A1_2_T5()
        {
            RunFile("S12.10_A1.2_T5.js");
        }

        [Test(Description = "Using \"with\" statement within function constructor, leading to normal completition")]
        public void S12_10_A1_3_T1()
        {
            RunFile("S12.10_A1.3_T1.js");
        }

        [Test(Description = "Using \"with\" statement within function constructor, leading to normal completition by \"return\"")]
        public void S12_10_A1_3_T2()
        {
            RunFile("S12.10_A1.3_T2.js");
        }

        [Test(Description = "Using \"with\" statement within function constructor, leading to normal completition by \"return\"")]
        public void S12_10_A1_3_T3()
        {
            RunFile("S12.10_A1.3_T3.js");
        }

        [Test(Description = "Using \"with\" statement within function constructor, leading to completition by exception")]
        public void S12_10_A1_3_T4()
        {
            RunFile("S12.10_A1.3_T4.js");
        }

        [Test(Description = "Using \"with\" statement within function constructor, leading to completition by exception")]
        public void S12_10_A1_3_T5()
        {
            RunFile("S12.10_A1.3_T5.js");
        }

        [Test(Description = "Using \"with\" statement within iteration statement, leading to normal completion")]
        public void S12_10_A1_4_T1()
        {
            RunFile("S12.10_A1.4_T1.js");
        }

        [Test(Description = "Using \"with\" statement within iteration statement, leading to completion by exception")]
        public void S12_10_A1_4_T2()
        {
            RunFile("S12.10_A1.4_T2.js");
        }

        [Test(Description = "Using \"with\" statement within iteration statement, leading to completion by exception")]
        public void S12_10_A1_4_T3()
        {
            RunFile("S12.10_A1.4_T3.js");
        }

        [Test(Description = "Using \"with\" statement within iteration statement, leading to completion by break")]
        public void S12_10_A1_4_T4()
        {
            RunFile("S12.10_A1.4_T4.js");
        }

        [Test(Description = "Using \"with\" statement within iteration statement, leading to completion by break")]
        public void S12_10_A1_4_T5()
        {
            RunFile("S12.10_A1.4_T5.js");
        }

        [Test(Description = "Using \"with\" statement within \"for-in\" statement, leading to normal completion")]
        public void S12_10_A1_5_T1()
        {
            RunFile("S12.10_A1.5_T1.js");
        }

        [Test(Description = "Using \"with\" statement within \"for-in\" statement, leading to completion by exception")]
        public void S12_10_A1_5_T2()
        {
            RunFile("S12.10_A1.5_T2.js");
        }

        [Test(Description = "Using \"with\" statement within \"for-in\" statement, leading to completion by exception")]
        public void S12_10_A1_5_T3()
        {
            RunFile("S12.10_A1.5_T3.js");
        }

        [Test(Description = "Using \"with\" statement within \"for-in\" statement, leading to completion by break")]
        public void S12_10_A1_5_T4()
        {
            RunFile("S12.10_A1.5_T4.js");
        }

        [Test(Description = "Using \"with\" statement within \"for-in\" statement, leading to completion by break")]
        public void S12_10_A1_5_T5()
        {
            RunFile("S12.10_A1.5_T5.js");
        }

        [Test(Description = "Using \"with\" statement within another \"with\" statement, leading to normal completion")]
        public void S12_10_A1_6_T1()
        {
            RunFile("S12.10_A1.6_T1.js");
        }

        [Test(Description = "Using \"with\" statement within another \"with\" statement, leading to completion by exception")]
        public void S12_10_A1_6_T2()
        {
            RunFile("S12.10_A1.6_T2.js");
        }

        [Test(Description = "Using \"with\" statement within another \"with\" statement, leading to completion by exception")]
        public void S12_10_A1_6_T3()
        {
            RunFile("S12.10_A1.6_T3.js");
        }

        [Test(Description = "Calling a function within \"with\" statement declared within the statement, leading to normal completion")]
        public void S12_10_A1_7_T1()
        {
            RunFile("S12.10_A1.7_T1.js");
        }

        [Test(Description = "Calling a function within \"with\" statement declared within the statement, leading to normal completion by \"return\"")]
        public void S12_10_A1_7_T2()
        {
            RunFile("S12.10_A1.7_T2.js");
        }

        [Test(Description = "Calling a function within \"with\" statement declared within the statement, leading to normal completion by \"return\"")]
        public void S12_10_A1_7_T3()
        {
            RunFile("S12.10_A1.7_T3.js");
        }

        [Test(Description = "Calling a function within \"with\" statement declared within the statement, leading to completion by exception")]
        public void S12_10_A1_7_T4()
        {
            RunFile("S12.10_A1.7_T4.js");
        }

        [Test(Description = "Calling a function within \"with\" statement declared within the statement, leading to completion by exception")]
        public void S12_10_A1_7_T5()
        {
            RunFile("S12.10_A1.7_T5.js");
        }

        [Test(Description = "Declaring function constructor within \"with\" statement, leading to normal completion")]
        public void S12_10_A1_8_T1()
        {
            RunFile("S12.10_A1.8_T1.js");
        }

        [Test(Description = "Declaring function constructor within \"with\" statement, leading to normal completion by \"return\"")]
        public void S12_10_A1_8_T2()
        {
            RunFile("S12.10_A1.8_T2.js");
        }

        [Test(Description = "Declaring function constructor within \"with\" statement, leading to normal completion by \"return\"")]
        public void S12_10_A1_8_T3()
        {
            RunFile("S12.10_A1.8_T3.js");
        }

        [Test(Description = "Declaring function constructor within \"with\" statement, leading to completion by exception")]
        public void S12_10_A1_8_T4()
        {
            RunFile("S12.10_A1.8_T4.js");
        }

        [Test(Description = "Declaring function constructor within \"with\" statement, leading to completion by exception")]
        public void S12_10_A1_8_T5()
        {
            RunFile("S12.10_A1.8_T5.js");
        }

        [Test(Description = "Using \"for-in\" statement within \"with\" statement, leading to normal completion")]
        public void S12_10_A1_9_T1()
        {
            RunFile("S12.10_A1.9_T1.js");
        }

        [Test(Description = "Using \"for-in\" statement within \"with\" statement, leading to completion by break")]
        public void S12_10_A1_9_T2()
        {
            RunFile("S12.10_A1.9_T2.js");
        }

        [Test(Description = "Using \"for-in\" statement within \"with\" statement, leading to completion by break")]
        public void S12_10_A1_9_T3()
        {
            RunFile("S12.10_A1.9_T3.js");
        }

        [Test(Description = "Using iteration statement within \"with\" statement, leading to normal completion")]
        public void S12_10_A3_10_T1()
        {
            RunFile("S12.10_A3.10_T1.js");
        }

        [Test(Description = "Using iteration statement within \"with\" statement, leading completion by exception")]
        public void S12_10_A3_10_T2()
        {
            RunFile("S12.10_A3.10_T2.js");
        }

        [Test(Description = "Using iteration statement within \"with\" statement, leading completion by exception")]
        public void S12_10_A3_10_T3()
        {
            RunFile("S12.10_A3.10_T3.js");
        }

        [Test(Description = "Using iteration statement within \"with\" statement, leading completion be break")]
        public void S12_10_A3_10_T4()
        {
            RunFile("S12.10_A3.10_T4.js");
        }

        [Test(Description = "Using iteration statement within \"with\" statement, leading completion be break")]
        public void S12_10_A3_10_T5()
        {
            RunFile("S12.10_A3.10_T5.js");
        }

        [Test(Description = "Calling a function within \"with\" statement declared without the statement, leading to normal completion")]
        public void S12_10_A3_11_T1()
        {
            RunFile("S12.10_A3.11_T1.js");
        }

        [Test(Description = "Calling a function within \"with\" statement declared without the statement, leading to normal completion by \"return\"")]
        public void S12_10_A3_11_T2()
        {
            RunFile("S12.10_A3.11_T2.js");
        }

        [Test(Description = "Calling a function within \"with\" statement declared without the statement, leading to normal completion by \"return\"")]
        public void S12_10_A3_11_T3()
        {
            RunFile("S12.10_A3.11_T3.js");
        }

        [Test(Description = "Calling a function within \"with\" statement declared without the statement, leading to completion by exception")]
        public void S12_10_A3_11_T4()
        {
            RunFile("S12.10_A3.11_T4.js");
        }

        [Test(Description = "Calling a function within \"with\" statement declared without the statement, leading to completion by exception")]
        public void S12_10_A3_11_T5()
        {
            RunFile("S12.10_A3.11_T5.js");
        }

        [Test(Description = "Calling a function without \"with\" statement declared within the statement, leading to normal completion")]
        public void S12_10_A3_12_T1()
        {
            RunFile("S12.10_A3.12_T1.js");
        }

        [Test(Description = "Calling a function without \"with\" statement declared within the statement, leading to normal completion by \"return\"")]
        public void S12_10_A3_12_T2()
        {
            RunFile("S12.10_A3.12_T2.js");
        }

        [Test(Description = "Calling a function without \"with\" statement declared within the statement, leading to normal completion by \"return\"")]
        public void S12_10_A3_12_T3()
        {
            RunFile("S12.10_A3.12_T3.js");
        }

        [Test(Description = "Calling a function without \"with\" statement declared within the statement, leading to completion by exception")]
        public void S12_10_A3_12_T4()
        {
            RunFile("S12.10_A3.12_T4.js");
        }

        [Test(Description = "Calling a function without \"with\" statement declared within the statement, leading to completion by exception")]
        public void S12_10_A3_12_T5()
        {
            RunFile("S12.10_A3.12_T5.js");
        }

        [Test(Description = "Using \"with\" statement within global context - normal completion")]
        public void S12_10_A3_1_T1()
        {
            RunFile("S12.10_A3.1_T1.js");
        }

        [Test(Description = "Using \"with\" statement within global context, leading to completion by exception")]
        public void S12_10_A3_1_T2()
        {
            RunFile("S12.10_A3.1_T2.js");
        }

        [Test(Description = "Using \"with\" statement within global context, leading to completion by exception")]
        public void S12_10_A3_1_T3()
        {
            RunFile("S12.10_A3.1_T3.js");
        }

        [Test(Description = "Declaring \"with\" statement within a function body, leading to normal completion")]
        public void S12_10_A3_2_T1()
        {
            RunFile("S12.10_A3.2_T1.js");
        }

        [Test(Description = "Declaring \"with\" statement within a function body, leading to normal completion by \"return\"")]
        public void S12_10_A3_2_T2()
        {
            RunFile("S12.10_A3.2_T2.js");
        }

        [Test(Description = "Declaring \"with\" statement within a function body, leading to normal completion by \"return\"")]
        public void S12_10_A3_2_T3()
        {
            RunFile("S12.10_A3.2_T3.js");
        }

        [Test(Description = "Declaring \"with\" statement within a function body, leading to completion by exception")]
        public void S12_10_A3_2_T4()
        {
            RunFile("S12.10_A3.2_T4.js");
        }

        [Test(Description = "Declaring \"with\" statement within a function body, leading to completion by exception")]
        public void S12_10_A3_2_T5()
        {
            RunFile("S12.10_A3.2_T5.js");
        }

        [Test(Description = "Declaring \"with\" statement within a function constructor, leading to normal completion")]
        public void S12_10_A3_3_T1()
        {
            RunFile("S12.10_A3.3_T1.js");
        }

        [Test(Description = "Declaring \"with\" statement within a function constructor, leading to normal completion by \"return\"")]
        public void S12_10_A3_3_T2()
        {
            RunFile("S12.10_A3.3_T2.js");
        }

        [Test(Description = "Declaring \"with\" statement within a function constructor, leading to normal completion by \"return\"")]
        public void S12_10_A3_3_T3()
        {
            RunFile("S12.10_A3.3_T3.js");
        }

        [Test(Description = "Declaring \"with\" statement within a function constructor, leading to completion by exception")]
        public void S12_10_A3_3_T4()
        {
            RunFile("S12.10_A3.3_T4.js");
        }

        [Test(Description = "Using \"with\" statement within iteration statement, leading to normal completion")]
        public void S12_10_A3_4_T1()
        {
            RunFile("S12.10_A3.4_T1.js");
        }

        [Test(Description = "Using \"with\" statement within iteration statement, leading to completion by exception")]
        public void S12_10_A3_4_T2()
        {
            RunFile("S12.10_A3.4_T2.js");
        }

        [Test(Description = "Using \"with\" statement within iteration statement, leading to completion by exception")]
        public void S12_10_A3_4_T3()
        {
            RunFile("S12.10_A3.4_T3.js");
        }

        [Test(Description = "Using \"with\" statement within iteration statement, leading to completion by break")]
        public void S12_10_A3_4_T4()
        {
            RunFile("S12.10_A3.4_T4.js");
        }

        [Test(Description = "Using \"with\" statement within iteration statement, leading to completion by break")]
        public void S12_10_A3_4_T5()
        {
            RunFile("S12.10_A3.4_T5.js");
        }

        [Test(Description = "Using \"with\" statement within \"for-in\" statement, leading to normal completion")]
        public void S12_10_A3_5_T1()
        {
            RunFile("S12.10_A3.5_T1.js");
        }

        [Test(Description = "Using \"with\" statement within \"for-in\" statement, leading to completion by exception")]
        public void S12_10_A3_5_T2()
        {
            RunFile("S12.10_A3.5_T2.js");
        }

        [Test(Description = "Using \"with\" statement within \"for-in\" statement, leading to completion by exception")]
        public void S12_10_A3_5_T3()
        {
            RunFile("S12.10_A3.5_T3.js");
        }

        [Test(Description = "Using \"with\" statement within \"for-in\" statement, leading to completion by break")]
        public void S12_10_A3_5_T4()
        {
            RunFile("S12.10_A3.5_T4.js");
        }

        [Test(Description = "Using \"with\" statement within \"for-in\" statement, leading to completion by break")]
        public void S12_10_A3_5_T5()
        {
            RunFile("S12.10_A3.5_T5.js");
        }

        [Test(Description = "Using \"with\" statement within another \"with\" statement, leading to normal completion")]
        public void S12_10_A3_6_T1()
        {
            RunFile("S12.10_A3.6_T1.js");
        }

        [Test(Description = "Using \"with\" statement within another \"with\" statement, leading to completion by exception")]
        public void S12_10_A3_6_T2()
        {
            RunFile("S12.10_A3.6_T2.js");
        }

        [Test(Description = "Using \"with\" statement within another \"with\" statement, leading to completion by exception")]
        public void S12_10_A3_6_T3()
        {
            RunFile("S12.10_A3.6_T3.js");
        }

        [Test(Description = "Declaring and calling a function within \"with\" statement, leading to normal completion")]
        public void S12_10_A3_7_T1()
        {
            RunFile("S12.10_A3.7_T1.js");
        }

        [Test(Description = "Declaring and calling a function within \"with\" statement, leading to normal completion by \"return\"")]
        public void S12_10_A3_7_T2()
        {
            RunFile("S12.10_A3.7_T2.js");
        }

        [Test(Description = "Declaring and calling a function within \"with\" statement, leading to normal completion by \"return\"")]
        public void S12_10_A3_7_T3()
        {
            RunFile("S12.10_A3.7_T3.js");
        }

        [Test(Description = "Declaring and calling a function within \"with\" statement, leading to completion by exception")]
        public void S12_10_A3_7_T4()
        {
            RunFile("S12.10_A3.7_T4.js");
        }

        [Test(Description = "Declaring and calling a function within \"with\" statement, leading to completion by exception")]
        public void S12_10_A3_7_T5()
        {
            RunFile("S12.10_A3.7_T5.js");
        }

        [Test(Description = "Declaring function constructor within \"with\" statement, leading to normal completion")]
        public void S12_10_A3_8_T1()
        {
            RunFile("S12.10_A3.8_T1.js");
        }

        [Test(Description = "Declaring function constructor within \"with\" statement, leading to normal completion by \"return\"")]
        public void S12_10_A3_8_T2()
        {
            RunFile("S12.10_A3.8_T2.js");
        }

        [Test(Description = "Declaring function constructor within \"with\" statement, leading to normal completion by \"return\"")]
        public void S12_10_A3_8_T3()
        {
            RunFile("S12.10_A3.8_T3.js");
        }

        [Test(Description = "Declaring function constructor within \"with\" statement, leading to completion by exception")]
        public void S12_10_A3_8_T4()
        {
            RunFile("S12.10_A3.8_T4.js");
        }

        [Test(Description = "Declaring function constructor within \"with\" statement, leading to completion by exception")]
        public void S12_10_A3_8_T5()
        {
            RunFile("S12.10_A3.8_T5.js");
        }

        [Test(Description = "Using \"for-in\" statement within \"with\" statement, leading to normal completion")]
        public void S12_10_A3_9_T1()
        {
            RunFile("S12.10_A3.9_T1.js");
        }

        [Test(Description = "Using \"for-in\" statement within \"with\" statement, leading to completion by break")]
        public void S12_10_A3_9_T2()
        {
            RunFile("S12.10_A3.9_T2.js");
        }

        [Test(Description = "Using \"for-in\" statement within \"with\" statement, leading to completion by break")]
        public void S12_10_A3_9_T3()
        {
            RunFile("S12.10_A3.9_T3.js");
        }

        [Test(Description = "Changing string property")]
        public void S12_10_A4_T1()
        {
            RunFile("S12.10_A4_T1.js");
        }

        [Test(Description = "Changing number property")]
        public void S12_10_A4_T2()
        {
            RunFile("S12.10_A4_T2.js");
        }

        [Test(Description = "Changing boolean property")]
        public void S12_10_A4_T3()
        {
            RunFile("S12.10_A4_T3.js");
        }

        [Test(Description = "Changing object property")]
        public void S12_10_A4_T4()
        {
            RunFile("S12.10_A4_T4.js");
        }

        [Test(Description = "Changing array property")]
        public void S12_10_A4_T5()
        {
            RunFile("S12.10_A4_T5.js");
        }

        [Test(Description = "Changing function property")]
        public void S12_10_A4_T6()
        {
            RunFile("S12.10_A4_T6.js");
        }

        [Test(Description = "Deleting string property")]
        public void S12_10_A5_T1()
        {
            RunFile("S12.10_A5_T1.js");
        }

        [Test(Description = "Deleting number property")]
        public void S12_10_A5_T2()
        {
            RunFile("S12.10_A5_T2.js");
        }

        [Test(Description = "Deleting boolean property")]
        public void S12_10_A5_T3()
        {
            RunFile("S12.10_A5_T3.js");
        }

        [Test(Description = "Deleting object property")]
        public void S12_10_A5_T4()
        {
            RunFile("S12.10_A5_T4.js");
        }

        [Test(Description = "Deleting array property")]
        public void S12_10_A5_T5()
        {
            RunFile("S12.10_A5_T5.js");
        }

        [Test(Description = "Deleting function property")]
        public void S12_10_A5_T6()
        {
            RunFile("S12.10_A5_T6.js");
        }
    }
}
